packages/repository-json-schema/test/integration/build-schema.integration.ts

```diff
@@ -457,22 +457,9 @@ describe('build-schema', () => {
           });
           expect(customerSchema.properties).to.deepEqual({
             id: {type: 'number'},
-            orders: {
-              type: 'array',
-              items: {$ref: '#/definitions/Order'},
-            },
-          });
-          expect(customerSchema.definitions).to.deepEqual({
```

Comment:
I feel we should rework this check instead of removing it.

How about the following assertion?

```ts
expect(Object.keys(customerSchema.definitions)).to.not.containEql('Order');
```

Reply:
Hi @bajtos thanks for your comments.
The original tests were added and changed accordantly.
Code was reformatted using lint (npm run lint:fix).